### PR TITLE
CEDS-1206 Shut a MUCR journey

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/controllers/DisassociateDucrController.scala [hunk 1]
```diff
@@ -19,6 +19,7 @@ package controllers
 import config.AppConfig
 import controllers.actions.{AuthAction, JourneyAction}
 import controllers.util.CacheIdGenerator.movementCacheId
+import controllers.util.FlashKeys
```
Comment (ed0906) [inline app/controllers/DisassociateDucrController.scala]: Is this required? seems its not used

Comment (MaciejRewera) [reply app/controllers/DisassociateDucrController.scala]: It is used when submitting.

Comment (ed0906) [reply app/controllers/DisassociateDucrController.scala]: Ah you've changed the package

File: app/controllers/util/FlashKeys.scala [hunk 2]
```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  */
 
-package controllers
+package controllers.util
```
Comment (ed0906) [inline app/controllers/util/FlashKeys.scala]: I think we can come up with a better package for this than `util`

Given Flash is all about data storage how about `storage`/`persistance`

File: app/views/shut_mucr_confirmation.scala.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+@this(main_template: views.html.main_template)
+
+@()(implicit appConfig: AppConfig, request: Request[_], flash: Flash, messages: Messages)
+
+@main_template(
+  title = messages("shutMucrConfirmation.title"),
+  appConfig = appConfig
+) {
+
+  <div class="govuk-box-highlight">
```
Comment (ed0906) [inline app/views/shut_mucr_confirmation.scala.html]: Maybe we need a component for this summary given we're adding it in for a second time, what are your thoughts?

File: app/controllers/util/FlashKeys.scala (hunk 2 above)
Comment (MaciejRewera) [reply app/controllers/util/FlashKeys.scala]: DONE

File: app/views/shut_mucr_confirmation.scala.html (hunk 3 above)
Comment (MaciejRewera) [reply app/views/shut_mucr_confirmation.scala.html]: This is good idea. I'll add another commit with it.

**Pull request merged**